.github/workflows/readme.yml

```diff
@@ -0,0 +1,15 @@
+name: ci
```

Comment:
I think you have forgotten to rename.

Anyway, I propose to add `verify-readmes` target to `ci` in `Makfile` and get rid of this GitHub workflow.